src/global/utils/RandomUtil.java

```diff
@@ -106,9 +107,19 @@ public static boolean frequenciesAreRandom(int[] frequencies, int numBins) {
         return (count * 1.0) / numBins >= ACCEPTANCE_RATE;
     }
 
+    public static boolean frequenciesAreRandom(int[] frequencies, double percentageTrue) {
```

Comment:
Why can't we use the current function with numBins = 2?

Reply:
That is because we only need to really test the one bin, and the error acceptance is much lower for booleans than it is for everything else.